src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -154,9 +154,9 @@ private boolean isNeverSuppressCheck(TreeWalkerAuditEvent event) {
     }
 
     /**
-     * Whether line match.
+     * Is matching by lineNo.
      *
-     * @param event event to process.
+     * @param event {@code TreeWalkerAuditEvent} object
      * @return true if line and column are matching or not set.
```

Comment:
`@return true if line and column are matching or not set` - it doesn't check column
`... or not set` - if not set then line is 0, and it will be compared by lineMatching() anyway